test/test_rules_against_dialect_fixtures.sh

```diff
@@ -3,6 +3,11 @@
 echo "This script will run 'sqlfluff fix -f' on all our dialect fixtures"
 echo "to look for critical errors."
 echo "WARNING this will change the fixtures so do not commit these changes!"
+function rest_git_changes(){
+    echo "Attempting to reset Git Changes"
+    # If this doesnt work for some reason just ignore it
+    git checkout -- test/fixtures/dialects/ || true
+}
```

Comment:
I think going with the python version I purposed above is prolly better, but this little bonus might save a little shock for someone like me who totally ignored the warnings before running this the first time.